pages/common/seq.md

```diff
@@ -0,0 +1,15 @@
+# seq
+
+> Output a sequence of numbers to stdout
```

Comment:
Although this is technically correct, a simpler "Print a sequence of numbers" may be more immediately intuitive for beginners. What do you think?